en/micro-integrator/docs/use-cases/examples/sequence_examples/using-fault-sequences.md

```diff
@@ -11,8 +11,8 @@ Following are the integration artifacts that we can used to implement this scena
 
 -   Proxy service:
     ```xml
-    <proxy xmlns="http://ws.apache.org/ns/synapse" name="FaultTestProxy" startOnLoad="true" transports="http https">
-        <target>
+    <proxy name="FaultTestProxy" startOnLoad="true" statistics="enable" trace="enable" transports="http https" xmlns="http://ws.apache.org/ns/synapse">
```

Comment:
we can skip enabling trace and stat by default

Reply:
Fixed.